index.html

```diff
@@ -12,17 +12,15 @@
           <b>English: </b> <a href="http://www.clojurebridge.org/">ClojureBridge</a>
           is a free Clojure programming workshop for women* -
           especially for those with little to no programming experience.<br>
-          <b>Next workshop: June 17 and 18, 2016.</b><br>
-          Applications are closed.
+          <b>Next workshop: presumably January 2017</b><br>
```

Comment:
I think the date I've heard floating around is November/Dec (before xmas) :) But I suppose we'll come up with a date (or a rough range) at retro maybe, and we can change it then.